code/gui/camera.cpp

```diff
@@ -202,10 +202,13 @@ int getCameraIndex(const QCameraInfo &info) {
 
 void CameraDisplay::setVisible(bool visible) {
     if (visible) {
-        int camera_index = -1;
+        int camera_index = 0;
         if (!QCameraInfo::availableCameras().empty()) {
             camera_index = getCameraIndex(QCameraInfo::availableCameras()[0]);
         }
+        if (camera_index >= 10) {
```

Comment:
I know I made this change but can you move the `if (camera_index >= 10) camera_index = 0` hack into `getCameraIndex`?